resources/views/admin/checkin/trip.blade.php

```diff
@@ -126,6 +126,13 @@ class="traindelay">+{{ $stopover->arrival_real->diffInMinutes($stopover->arrival
                                         Mastodon
                                     </label>
                                 </div>
+                                <div class="form-check">
+                                    <input class="form-check-input" type="checkbox" id="shouldChain_check"
+                                           name="shouldChain_check" {{$user->mastodonUrl && \App\Http\Controllers\Backend\Social\MastodonController::getLastSavedPostIdFromUserStatuses($user) ? '' : 'disabled'}}>
+                                    <label class="form-check-label" for="shouldChain_check">
+                                        Toot Chaining
```

Comment:
Can this be added to the translations?

Reply:
In der Admin-Checkin-UI wird gar nichts übersetzt.

Reply:
Oh, sorry. Didn't pay attention to the fact that this is in the admin panel.